README.md

```diff
@@ -35,14 +47,29 @@ router.get('/', function (req, res) {
 });
 ```
 
-`index.html`
-```html
-<html>
-	<head></head>
-	<body>
-		<div id="app"><%- content %></div>
-	</body>
-</html>
+### Option: Pass jsx
+
+`index.js`:
+
+```javascript
+var router = require('express')();
+var reactExpressMiddleware = require('react-express-middleware');
+var PageWrapper = require('./page-wrapper.jsx');
+var Container = require('./container.jsx');
+
+router.use(reactExpressMiddleware({
+	element: 'app'
+}));
+router.get('/', function (req, res) {
+	var RenderComponent = (
+		<PageWrapper>
+			<Container name={res.locals.name} />
+			// <Container name="Lorelai" />
+		</PageWrapper>
+	);
+
+	res.renderReactComponent(RenderComponent);
+});
 ```
 
 **Note:** The module does not specify a react dependency so you can depend on whatever react version you want.  We only require greater than React 0.14.0
```

Comment:
Now that there is more to this readme, I think this note should be moved up toward the top.  maybe next to a line that has an npm install command for the peer deps: `npm i --save react react-dom`